src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.Designer.cs

```diff
@@ -117,7 +117,7 @@ internal class NuGetCommand {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to FindBy added to a store client certificate source..
```

Comment:
Should we include changes in this file? For me it looks different from `Lock` in other files.

Reply:
This file is automatically generated. I'd say let's keep the change